pom.xml

```diff
@@ -12,6 +12,12 @@
             <version>5.9.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <version>RELEASE</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
dodałeś zależność, ale nie widzę, żebyś ją gdziekolwiek używał - jesli nie potrzeba, to usuń.